__tests__/app.test.js

```diff
@@ -95,3 +95,33 @@ describe('GET /api/articles/:article_id', () => {
                 })
             })
 })
+describe('/api/articles/:article_id/comments', () => {
+    test('200: should return all the comments for the specific article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments).toHaveLength(11);
+            expect(body.comments[0].article_id).toBe(1);
```

Comment:
Generally it's always best to iterate through the results and check every one, rather than just checking the one. This test could pass if the first element happens to have an id of 1, but the rest don't.